crm_operating_unit/models/crm_lead.py

```diff
@@ -3,14 +3,36 @@
 # Jordi Ballester Alomar
 # © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
-from openerp import fields, models
+from openerp import api, fields, models, _
+from openerp.exceptions import UserError
 
 
 class CRMLead(models.Model):
 
     _inherit = 'crm.lead'
 
+    @api.model
```

Comment:
You can't add this code here, as there's no dependency to the other module. Do it in the proper module.